com.corifeus.onenote.yml

```diff
@@ -0,0 +1,71 @@
+app-id: com.corifeus.onenote
+runtime: org.freedesktop.Platform
+runtime-version: '20.08'
+branch: stable
+sdk: org.freedesktop.Sdk
+base: org.electronjs.Electron2.BaseApp
+base-version: stable
+command: run.sh
+separate-locales: false
+finish-args:
+  - --share=ipc
+  - --socket=x11
+  - --socket=pulseaudio
+  - --share=network
+build-options:
+  cflags: -O2 -g
+  cxxflags: -O2 -g
+  env:
+    NPM_CONFIG_LOGLEVEL: info
+sdk-extensions:
+  - org.freedesktop.Sdk.Extension.node12
+modules:
+  - name: zypak
+    sources:
+      - type: git
+        url: https://github.com/refi64/zypak
+        tag: v2020.02
+  - name: p3x-onenote
+    build-options:
+      append-path: /usr/lib/sdk/node12/bin
+      env:
+        # Need this for electron-download to find the cached electron binary
+        electron_config_cache: /run/build/p3x-onenote/npm-cache/electron_cache
+        ELECTRON_CACHE: /run/build/p3x-onenote/npm-cache
+    buildsystem: simple
+    sources:
+      - type: git
+        url: https://github.com/patrikx3/onenote.git
+        commit: dba0e5e
+        dest: main
+      - type: file
+        path: package-lock.json
+        dest: main/
+      - type: patch
+        path: package.patch
+        options:
+          - "-dmain"
+      - type: file
+        path: com.corifeus.onenote.desktop
+      - type: file
+        path: com.corifeus.onenote.png
+      - type: file
+        path: com.corifeus.onenote.appdata.xml
```

Comment:
New appdata files should use metainfo instead. https://www.freedesktop.org/software/appstream/docs/chap-Quickstart.html